.github/workflows/e2e.yml

```diff
@@ -160,19 +160,22 @@ jobs:
         working-directory: opencollective-frontend
         run: CYPRESS_INSTALL_BINARY=0 npm ci --prefer-offline --no-audit
 
+      - name: Set commit hash (frontend)
+        working-directory: opencollective-frontend
+        run: echo "FRONTEND_COMMIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
+
       - name: Restore build (frontend)
         uses: actions/cache@v1
         id: next-build
         with:
           path: opencollective-frontend/dist
-          key: ${{ runner.os }}-next-build-${{ hashFiles('opencollective-frontend/**') }}
+          key: ${{ runner.os }}-next-build-${{ env.FRONTEND_COMMIT_HASH }}
 
       - name: Restore .next cache (frontend)
-        if: steps.next-build.outputs.cache-hit != 'true'
```

Comment:
Why are you removing this?

Reply:
This should not be changed if there is not good reason for it.

Reply:
If we're able to restore the build, no reason to restore .next cache.

Reply:
@znarf : If we don't remove this when we restart the github action builds it will fail with; https://github.com/opencollective/opencollective-api/runs/3077008271. My understanding is that the reason it was working before is actually because even when the build is restarted (with no changes) the build step will always run. To demonstrate this I have created the commit; 6e37d9c (which is what we had before this PR). I've manually restarted the build for this commit. Note that in that build run (https://github.com/opencollective/opencollective-api/runs/3077512247), the cache was not found; although we expect it to be found since it already ran the first time. 

My thinking is that when we restart the build the value `hashFiles('opencollective-frontend/**')` is different that what we stored the cache for. The previous build steps post cache operation stores the cache value but at that time the whole `opencollective-frontend` directory undergoes changes and hence the hash value is not accurate. I couldn't find any previous builds where this build step was skipped tbh. So this is why I thought that this cache hit should be removed. Does that make sense?

Reply:
@SudharakaP Thank you for the explanations.

I'm afraid that if we remove the "skip", some steps are losing their purpose.

We'll need to look deeper into this before merging.